### ISSUE TITLE
More of a feature request than an issue

### ISSUE THREAD

**Issue by srcr** [countercept/snake-tail#2]
Currently if you try to upload a file, for example, and the file already exists the snake-tail tools prints the API error from the web server and does an exit (0) ideally you would want to reflect this error in the exit code when the tool exits.

```
$ snake-tail upload file a.exe ; echo $?
Status: Error
Message: upload/file - file already exists for given sha256 digest
0
```

**Comment by alexkornitzer** [comment]
Agreed, snake-tail is a little out of date, when I get round to bringing it up to scratch I will handle the above.
